es-index.html

```diff
@@ -0,0 +1,44 @@
+--- title: "Jamulus ‒ Toca música online. Con amig@s. Gratis." lang: "es" permalink: / layout: mainhomepage ldjAppCategory: "Music" ldjSlogan: "Toca música online. Con amig@s. Gratis." ldjCreator: "Volker Fischer y contribuidores" metadescription: "Jamulus es software de código abierto que permite a músicos actuar, ensayar o simplemente hacer jam sessions, en tiempo real a través de Internet." mAltProgIcon: "Jamulus icon" mTSlogan: "Toca música online. Con amig@s. Gratis." mTGetStartedNow: "¡Empieza ya!" mHTMLDownloadNow: 'Avanzado: <a href="https://sourceforge.net/projects/llcon/files/latest/download" target="_blank" id="dld_frm_sf" rel="noreferrer">descarga ahora</a>' ---
```

Comment:
I'm not sure if you can do it all in one line?

Reply:
Yep, my translation tool messed up the formatting - should be corrected now.

Reply:
Ok. The last `---` should probably also be in a new line.

Reply:
Yes, you're right... fixed!